test/test.js

```diff
@@ -2864,6 +2864,16 @@ exports['wrap EventEmitter (or jQuery) on handler'] = function (test) {
     });
 };
 
+exports['removing EventEmitter (or jQuery) listener on destruction'] = function(test) {
+    var ee = new EventEmitter();
+    var s = _('myevent', ee);
+
+    test.same(typeof ee._events.myevent, 'function');
```

Comment:
Checking private state like `_events` is brittle. You should listen to the 'removeListener' event instead, and make sure that is is emitted when you destroy the stream.